packages/create-instantsearch-app/src/cli/index.js

```diff
@@ -201,6 +201,89 @@ const getQuestions = ({ appName }) => ({
       when: ({ appId, apiKey, indexName }) =>
         attributesForFaceting.length === 0 && appId && apiKey && indexName,
     },
+    {
+      type: 'confirm',
+      name: 'useAutocomplete',
+      message: 'Use an autocomplete as a search box',
+      suffix: `\n  ${chalk.gray(
+        'Displays a list of suggested and recent searches as you type'
+      )}`,
+      default: true,
+      when: ({ libraryVersion, template }) => {
+        const templatePath = getTemplatePath(template);
+        const templateConfig = getAppTemplateConfig(templatePath);
+
+        const selectedLibraryVersion = libraryVersion;
+        const requiredLibraryVersion =
+          templateConfig.flags && templateConfig.flags.autocomplete;
+        const supportsAutocomplete =
+          selectedLibraryVersion &&
+          requiredLibraryVersion &&
+          semver.satisfies(selectedLibraryVersion, requiredLibraryVersion, {
+            includePrerelease: true,
+          });
+
+        return supportsAutocomplete;
+      },
+    },
+    {
+      type: 'input',
+      name: 'querySuggestionsIndexName',
+      message: 'Index name for Query Suggestions',
+      suffix: `\n  ${chalk.gray(
+        'Used to populate the autocomplete with suggested searches'
+      )}`,
+      default: 'instant_search_demo_query_suggestions',
+      when: ({ useAutocomplete }) => useAutocomplete,
+    },
+    {
+      type: 'list',
+      name: 'autocompleteLibraryVersion',
+      message: () => `Autocomplete version`,
+      choices: async () => {
+        const libraryName = '@algolia/autocomplete-js';
+
+        try {
+          const versions = await fetchLibraryVersions(libraryName);
+          const latestStableVersion = semver.maxSatisfying(versions, '*', {
```

Comment:
```suggestion
          const latestStableVersion = semver.maxSatisfying(versions, '1', {
```

maybe this should be configurable by the template somehow too? for now it's fine as I don't envision a major version in autocomplete any time soon

Reply:
Maybe we could just return `dist-tags.latest` from the npm API (with `fetchLibraryVersions`) then pass it to `semver.major` to get the accepted range?

Reply:
that's the same as `*`, no? I want to avoid v2 releasing and us forgetting that the template doesn't work with it or something. 

I don't think it's that big of a deal whichever way we go though